drone-webdriver/src/main/java/org/jboss/arquillian/drone/webdriver/window/Dimensions.java

```diff
@@ -0,0 +1,30 @@
+package org.jboss.arquillian.drone.webdriver.window;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class Dimensions {
+
+    public static final Pattern DIMENSIONS_PATTERN = Pattern.compile("([0-9]+)x([0-9]+)");
+
+    private static int width;
+    private static int height;
+
+    public static int getWidth() {
+        return width;
+    }
+
+    public static int getHeight() {
+        return height;
+    }
+
+    public static boolean isValidDimensionPattern(String dimensions) {
```

Comment:
Don't use it as a class with static methods - the values are not intended to be used as constants. Change it to normal object class with constructor and non-static methods